pkg/sentry/fsimpl/gofer/gofer.go

```diff
@@ -879,6 +947,10 @@ type dentry struct {
 	// children is protected by dirMu.
 	children map[string]*dentry
 
+	// If this dentry represents a directory, negativeChildrenCache cache
+	// names of negative children, negativeChildrenCache is protected by dirMu.
+	negativeChildrenCache *negativeNamesCache
```

Comment:
`negativeChildrenCache` should be a field by value, to avoid the allocation even when we start using the cache. AFAICT it only contains pointers and numbers.

Reply:
done